tests/test-env-py38.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
   - python=3.8
 
   # Datacube
-  - datacube>=1.8.8
+  - datacube<=1.8.9
```

Comment:
Why the change to `<=`, that seems a bit strange to me. I'd expect a minimum bound at least.

Reply:
Oh, because of this: https://github.com/opendatacube/datacube-core/issues/1412

Reply:
Ooooh, okay! I think I hit that today and didn't even realise it, I'll have to investigate further. Thanks

Reply:
Paul is aware of it too.